server/games/game.py

```diff
@@ -462,7 +406,7 @@ def host_left_lobby() -> bool:
                     await self.mark_invalid(ValidityState.MUTUAL_DRAW)
                     return
 
-                if len(self._results) == 0:
+                if not self._results:
```

Comment:
When is this `True`? Does this maybe rely on `collections.abc.Mapping.__len__` being nonzero or `GameResults.__len__` being nonzero? Or just `self._results` not being `None`? I am uncertain how trucity/casting to bool is handled for user-defined class if there is no `__bool__`.

Reply:
According to Python docs, in absence of `__bool__` truthiness is decided by `__len__` being non-zero. So, it's perfectly fine to idiomatically check for non-emptiness.

Reply:
Good to know, thanks!